modules/EvseV2G/charger/ISO15118_chargerImpl.hpp

```diff
@@ -10,7 +10,7 @@
 
 #include <generated/interfaces/ISO15118_charger/Implementation.hpp>
 
-#include "../EvseV2G.hpp"
```

Comment:
Why deleting `..`?

Reply:
relative paths in #include are generally not a good idea. The compiler flags already list the parent directory as place to look for include files, hence the .. isn't needed.

Main reason, I've used IDEs that try to be "helpful" and have automatically added unnecessary relative paths. It is habit to remove them. I don't know if there is any specific EVerest style guidance on whether .. in #include should be used or not.

Happy to revert that.

Reply:
Thanks for the explanation. Then we'll keep it that way.